packages/nvidia-container-toolkit/nvidia-container-toolkit.spec

```diff
@@ -14,13 +14,11 @@ License: Apache-2.0
 URL: https://%{goimport}
 
 Source0: https://%{goimport}/archive/v%{gover}/nvidia-container-toolkit-%{gover}.tar.gz
-Source1: nvidia-container-toolkit-config-k8s.toml
-Source2: nvidia-container-toolkit-config-ecs.toml
-Source3: nvidia-oci-hooks-json
-Source4: nvidia-gpu-devices.rules
-Source5: nvidia-container-toolkit-tmpfiles-ecs.conf
-Source6: nvidia-container-toolkit-tmpfiles-k8s.conf
-Source7: nvidia-container-toolkit-config-k8s
```

Comment:
The point of the non-templated version of the config file is to ensure that some valid configuration is present even if the downstream variant isn't configured with the corresponding settings. I don't think we want to drop it.